test/verify/check-machines

```diff
@@ -393,6 +393,48 @@ class TestMachines(MachineCase):
         b.wait_in_text("#vm-subVmTest1-disks-hda-target", "hda")
         b.wait_in_text("#vm-subVmTest1-disks-hdb-target", "hdb")
 
+    def testNetworks(self):
+        b = self.browser
+        m = self.machine
+
+        self.startVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        b.wait_in_text("body", "Virtual Machines")
+        b.wait_in_text("tbody tr th", "subVmTest1")
+
+        b.click("tbody tr th") # click on the row header
+        b.wait_present("#vm-subVmTest1-state")
+        b.wait_in_text("#vm-subVmTest1-state", "running")
+
+        b.wait_present("#vm-subVmTest1-networks") # wait for the tab
+        b.click("#vm-subVmTest1-networks") # open the "Networks" subtab
+
+        b.wait_in_text("#vm-subVmTest1-network-1-type", "network")
```

Comment:
This might fail. I think waiting for finishing the previous `b.click()` is needed via b.wait_present()